db/migrate/20210802074146_create_freaks.rb

```diff
@@ -5,7 +5,7 @@ def change
     create_table :freaks do |t|
       t.string :name
       t.string :description
-      t.date :photo
+      t.string :photo
```

Comment:
We need to remove this column from the freaks table, a photo is modeled as a separate table.

Reply:
Done!